pkg/ipam/ipam.go

```diff
@@ -153,7 +154,7 @@ func (ipam *IPAM) reserveLocalRoutes() {
 		return
 	}
 
-	routes, err := netlink.RouteList(nil, netlink.FAMILY_V4)
```

Comment:
:thinking: what if netlink changes the FAMILY_V4 to something else? Also, we have plenty of places that we are using `netlink.FAMILY_V4` which could bring confusion if this case is different than the others. Can't we have add a go tag to this file and have another one for macOS?

Reply:
In all fairness, I would be quite surprised if `AF_INET` changes, hence I thought it was just fine to do it this.
I didn't grep for all references to `netlink.FAMILY_V4`, so if we do have a few of these, I'm happy to do what you are suggesting, I was just thinking to do the simplest thing.

I am also not sure how come `netlink.RouteList` actually compiles on macOS. I'll update the PR with a version that uses build tags and stubs out this entire function.